packages/esm-patient-test-results-app/src/filter/filter-context.tsx

```diff
@@ -1,7 +1,7 @@
-import React, { createContext, useReducer, useEffect, useMemo, useState } from 'react';
+import React, { createContext, useReducer, useEffect, useMemo } from 'react';
 import { parseTime } from '../timeline/useTimelineData';
 import reducer from './filter-reducer';
-import { TreeNode } from './filter-set';
+import { TreeNode, FilterContextProps, FilterProviderProps, obsShape } from './filter.types';
```

Comment:
Hi @ZacButko! Shouldn't the file name be filter-types, since we have the same conventions at different places in the application?